src/main/java/io/github/pixee/security/Newlines.java

```diff
@@ -17,4 +17,14 @@ public static String stripAll(final String value) {
     }
     return value.replace("\r", "").replace("\n", "");
   }
+
+  /**
+   * Removes newlines from the given string, if any exist.
+   *
+   * @param value the given string to sanitize
+   * @return a {@link String} identical to the one given, without newline characters
+   */
+  public static String stripAll(final Object value) {
```

Comment:
Why `Object`? and `String.valueOf`?

Reply:
Because the least disruptive change to address the risk of log injection vulnerabilities is a solution that looks like this:

```diff
- log.info("hi {}", taint);
+ log.info("hi {}", stripNewlines(taint));
```

But `taint` isn't guaranteed to be a `String` in all situations, so this ease-of-use API can help with the code change delivery being guaranteed to be safe every time.